lib/gooddata/models/process.rb

```diff
@@ -201,7 +202,7 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
           end
 
         if res.keys.first == 'asyncTask'
-          res = JSON.parse(client.poll_on_code(res['asyncTask']['links']['poll'], options.merge(process: false)))
+          res = MultiJson.parse(client.poll_on_code(res['asyncTask']['links']['poll'], options.merge(process: false)))
```

Comment:
Refers to 'client' more than self (maybe move it to another class?) (FeatureEnvy)